LootLockerServerSDK/Source/LootLockerServerSDK/Public/ServerAPI/LootLockerServerMetadataRequest.h

```diff
@@ -97,7 +120,11 @@ struct FLootLockerServerMetadataEntry
      */
     UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
     TArray<FString> Tags;
-
+    /*
+     The access level set for this metadata entry. Valid values are game_api.read and game_api.write, though no values are required.
```

Comment:
This is not true (yet), nothing is game_api.write at the moment.

Reply:
Functionally yes, but technically no: You are still allowed to set game_api.write (at least as of yet). What I will do is add a note about different sources allowing only a subset of values.